typescript/10-space-age/space-age.ts

```diff
@@ -0,0 +1,23 @@
+type Planet = 'mercury' | 'venus' | 'earth' | 'mars' | 'jupiter' | 'saturn' | 'uranus' | 'neptune';
+type PlanetOrbitalPeriods = Record<Planet, number>;
+
+export function age(planet: Planet, seconds: number): number {
+	const EARTH_YEAR_SECONDS = 31557600;
+
+	const ORBITAL_PERIODS: PlanetOrbitalPeriods = {
```

Comment:
Why are these constants recalculated every time `age` is called?
(And why is this file indented with tabs?)

Reply:
Different tasks have different "Prettier" settings. I will pay attention to this